pkg/model/model.go

```diff
@@ -23,6 +23,7 @@ import (
 	"github.com/aws-controllers-k8s/code-generator/pkg/generate/templateset"
 	"github.com/aws-controllers-k8s/code-generator/pkg/names"
 	"github.com/aws-controllers-k8s/code-generator/pkg/util"
+	awssdkmodel "github.com/aws/aws-sdk-go/private/model/api"
```

Comment:
nit: `goimports` can help with separating internal from external imports

Reply:
I think `goimports` places it the same place, it was placed there by `gopls`. I tried deleting the line and by running `goimports -w ./pkg/model/model.go` I get the same result.

The sibling file
https://github.com/aws-controllers-k8s/code-generator/blob/2f11b23b1618466348fc850f9302c5ae3441580a/pkg/model/field.go#L19-L22 is also the same. But you're right, I think I've seen a convention where stdlib packages have one group, same-module has another group and all others have a third group.